### PR TITLE
I've fixed the notes and improved the derivation with Jacobians

### PR DESCRIPTION
**Pull request by wilkeraziz**

### CODE REVIEW

**Comment by philschulz** [comment]
That's awesome, thanks a lot! I once started the notes but never really looked at them again. I had even forgotten that they were on the branch. Do you think they should be on master or should we keep them behind the scenes for now?

File: modules/M6_DeepGenerativeModels/ExplainingReparametrisationGradients.tex [hunk 1]
```diff
@@ -28,28 +29,63 @@ \section{Derivation}
...
+\begin{subequations}\label{eq:h}
 \begin{align}
-Z &= h(x, \lambda) = (X - \mu) C^{-1} \\
-X &= h^{-1}(z, \lambda) = ZC + \mu
+z &= h(x, \lambda) = C^{-1} (x - \mu)  \\
+x &= h^{-1}(z, \lambda) = \mu + Cz ~ .
 \end{align}
-We thus get the following transformed density which is equivalent to $ q(\theta|\lambda) $, where we use $ |\cdot| $ to denote the absolute value function.
+\end{subequations}
+Note that $\phi(z)$ does not depend on $\lambda$ which were absorbed in the affine transformation---this in fact restricts the class of approximations $q(x|\lambda)$ to location-scale distributions.
```
Comment (philschulz) [inline modules/M6_DeepGenerativeModels/ExplainingReparametrisationGradients.tex]: Should be "was absorbed", right?

File: modules/M6_DeepGenerativeModels/ExplainingReparametrisationGradients.tex [hunk 2]
```diff
@@ -28,28 +29,63 @@ \section{Derivation}
...
+
+For the sake of generality we take $x$ and $z$ to be vector valued. Then we write $J_{h(x, \lambda)}$ to denote the Jacobian matrix of the transformation $h(x, \lambda)$, and $J_{h^{-1}(z, \lambda)}$ to denote the Jacobian matrix of the inverse transformation.\footnote{Recall that a Jacobian matrix $\mathbf J \triangleq J_{f(x)}$ of some vector value function $f(x)$ is such that $J_{i,j} = \pdv{x_j} f_i(x)$.}
```
Comment (philschulz) [inline modules/M6_DeepGenerativeModels/ExplainingReparametrisationGradients.tex]: "vector valued function" (you forgot the d)

**Comment by wilkeraziz** [comment]
I agree about generalising it. I used the same variables you already had because I thought you had a good reason for choosing them :p At some point I will merge this with some of my notes on ADVI and G-rep. I think we can already merge this with the master (even though we mean to improve it).

**Comment by wilkeraziz** [comment]
I'm merging the current version which should be a good prequel or sequel to module 6.

**Pull request merged**